compiler/sem/nilcheck.nim

```diff
@@ -358,7 +351,8 @@ proc namedSetsDebugInfo(ctx: NilCheckerContext, map: NilMap): string =
     result.add("} ")
   result.add("\n")
 
-proc namedMapAndSetsDebugInfo(ctx: NilCheckerContext, map: NilMap): string =
+proc namedMapAndSetsDebugInfo(ctx: NilCheckerContext,
+                              map: NilMap): string {.used.} =
```

Comment:
Non-blocker: should debug routines be marked as `{.deprecated: "do not use in final build".}`, similar to `astrepr`?

Reply:
Hm, yeah, that sounds like a good idea -- I'll change it.